WindowsServerDocs/administration/hybrid-cloud-print/hybrid-cloud-print-deploy.md

```diff
@@ -211,7 +211,7 @@ To enable authenticated communication with the HCP services, we need to create 3
     ![Print Server Cloud Print Deploy](../media/hybrid-cloud-print/PrintServer-CloudPrintDeploy.png)
 
     - Check the log file to see if there is any error:
-    `C:\Program Files\WindowsPowerShell\Modules\PublishCloudPrinter\1.0.0.0>notepad CloudPrintDeploy.log`
+    `C:\Program Files\WindowsPowerShell\Modules\PublishCloudPrinter\1.0.0.0\CloudPrintDeploy.log`
```

Comment:
Please note that the original text intends to show the reader how to open the log file in Notepad from within the directory folder
```
C:\Program Files\WindowsPowerShell\Modules\PublishCloudPrinter\1.0.0.0>
```
and to open the file `CloudPrintDeploy.log` there. Your suggested change will only work for those users who have configured Windows to open any `*.log` file in Notepad or a similar editor for viewing.

I do not recommend making this change.